### DISCUSSION TITLE
Bug ? Roles : Directus Folder

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#14974] [Archive]
Steps to reproduce : 

Remove all access to directus folder to a specific group.

They still have the button "All the files".  ( /admin/files/all )

In my opinion, that make no sense.  If we remove access to folder, why should they see "all items", but "without the name of folder" ?

**Comment by br41nslug** [comment]
Because directus folder are not related to what you'd consider filesystem folders, files can exist without being in a folder that's actually the default unless you explicitly specify a folder. I personally look at them more as `tags` than `folders` because people tend to have a lot of assumptions along with the word folder.

**Reply by nervusdm** [reply]
I do understand what you are saying, but it's  for me more about a bad pattern, without any offense for anyone.

Look at the directus_users. You can hide roles (this is pretty much like a folder), and users.

If you "hide folders", people should not be able to read "all files inside". That's the common understanding of how folders works.

Btw, tags already exists.

I believe it may require, one day, an upgrade.

**Reply by rijkvanzanten** [reply]
> Look at the directus_users. You can hide roles (this is pretty much like a folder), and users.

Funnily enough, your example here works the exact same way as files currently. If you disable read access to a role, you don't automatically disable read access to users.
